### ISSUE TITLE
[Accessibility] Missing label in Device Name textbox

### ISSUE THREAD

**Issue by davitol** [owncloud/core#24808]
### Steps to reproduce
1. Go to Personal Page menu, and scroll down to Devices section
### Expected behaviour

 Device Name textbox should have a label
### Actual behaviour

 Device Name textbox is not labered
### Server configuration

**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:"9.1.0.3","versionstring":"9.1.0 pre alpha","edition":"Enterprise"

**Updated from an older ownCloud or fresh install:**
Fresh

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** 
No

**Logs**

```

```
### Client configuration

**browser**

 Firefox

![screen shot 2016-05-24 at 13 12 58](https://cloud.githubusercontent.com/assets/10596560/15502094/28da0852-21b2-11e6-9dd8-cccb918e505c.png)

@owncloud/designers @ChristophWurst

**Comment by PVince81** [comment]
moved to more general UX group ticket: https://github.com/owncloud/core/issues/27578
